Util/Xamarin.Kotlin.BindingSupport/native/.idea/workspace.xml

```diff
@@ -1,110 +0,0 @@
-              </path>
-              <path>
-                <item name="" type="6a2764b6:ExternalProjectsStructure$RootNode" />
-                <item name="binding-support" type="f1a62948:ProjectNode" />
-                <item name="Source Sets" type="e897c970:GradleViewContributor$SourceSetsNode" />
-              </path>
-              <path>
-                <item name="" type="6a2764b6:ExternalProjectsStructure$RootNode" />
-                <item name="binding-support" type="f1a62948:ProjectNode" />
-                <item name="Tasks" type="e4a08cd1:TasksNode" />
-              </path>
-              <path>
-                <item name="" type="6a2764b6:ExternalProjectsStructure$RootNode" />
-                <item name="binding-support" type="f1a62948:ProjectNode" />
-                <item name="Tasks" type="e4a08cd1:TasksNode" />
-                <item name="build" type="c8890929:TasksNode$1" />
-              </path>
-            </expand>
-            <select />
-          </tree_state>
-        </projects_view>
-      </state>
-    </system>
-  </component>
-  <component name="ProjectId" id="1PRSN7irY32nr1xVPk76kAL48kQ" />
-  <component name="PropertiesComponent">
-    <property name="ASKED_ADD_EXTERNAL_FILES" value="true" />
-    <property name="ASKED_SHARE_PROJECT_CONFIGURATION_FILES" value="true" />
-    <property name="SHARE_PROJECT_CONFIGURATION_FILES" value="true" />
-    <property name="project.structure.last.edited" value="Modules" />
-    <property name="project.structure.proportion" value="0.0" />
-    <property name="project.structure.side.proportion" value="0.0" />
-  </component>
-  <component name="RunDashboard">
-    <option name="ruleStates">
-      <list>
-        <RuleState>
-          <option name="name" value="ConfigurationTypeDashboardGroupingRule" />
-        </RuleState>
-        <RuleState>
-          <option name="name" value="StatusDashboardGroupingRule" />
-        </RuleState>
-      </list>
-    </option>
-  </component>
-  <component name="RunManager" selected="Kotlin.xamarin.binding.kotlin.bindingsupport.ProgramKt">
-    <configuration name="native [jar]" type="GradleRunConfiguration" factoryName="Gradle">
-      <ExternalSystemSettings>
-        <option name="executionName" />
-        <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="externalSystemIdString" value="GRADLE" />
-        <option name="scriptParameters" />
-        <option name="taskDescriptions">
-          <list />
-        </option>
-        <option name="taskNames">
-          <list>
-            <option value="jar" />
-          </list>
-        </option>
-        <option name="vmOptions" />
-      </ExternalSystemSettings>
-      <GradleScriptDebugEnabled>true</GradleScriptDebugEnabled>
-      <method v="2" />
-    </configuration>
-    <configuration name="xamarin.binding.kotlin.bindingsupport.ProgramKt" type="JetRunConfigurationType" factoryName="Kotlin" nameIsGenerated="true">
```

Comment:
Could we keep these .idea files so that the dev does not have to configure the run targets for the app or the build target for the jar?
It somehow feels like it should be removed, but if you do, then intellij suddenly does not understand how to run the kotlin app.

Reply:
I do not know how those files got into master. They popped up suddenly 2-3 weeks ago and come from `Xamarin.Kotlin.BindingsSupport`. I was locally dragging it for weeks and had to take care of it.

All my PRIVATE project have `**/.idea` in `.gitignore`. Rider folder. Shouldn't we add those to `.gitignore`?

Reply:
I assumed those files slipped in by accident. My 1st decision was to keep em and take care not to commit or do anything. 
After few weeks I changed my mind and decided to remove them, though they were not mine. Maybe that was the mistake. Sorry.

Reply:
Yeah sounds like we should bring them back, can you revert the deletes of those please?